stetho/src/main/java/com/facebook/stetho/inspector/elements/ActivityNodeDescriptor.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+package com.facebook.stetho.inspector.elements;
+
+import android.app.Activity;
+import android.view.Window;
+
+final class ActivityNodeDescriptor extends ChainedNodeDescriptor<Activity> {
+  public ActivityNodeDescriptor() {
+  }
+
+  // TODO: do we have to worry about listening for when the Activity's Window transitions
```

Comment:
When an Activity's onCreate is called, it is attached to a window.    This is the point where you are inserting it into your structure.